src/components/navbar/navbar.jsx

```diff
@@ -0,0 +1,90 @@
+import React from 'react';
+import { NavLink } from 'react-router-dom';
+import { connect } from 'react-redux';
+
+import style from './navbar.module.css';
+import { logoutUserThunkCreator } from '../../redux/auth-reducer';
+
+function Navbar(props) {
+  const { login, path } = props;
+
+  return (
+    <div className={style.navbar_body}>
+      <nav>
+        <input type='checkbox' id={style.check} />
+        <label htmlFor={style.check} className={style.checkbtn}>
+          <i className='fas fa-bars'></i>
+        </label>
+        <NavLink className={style.logo_a} to='/'>
+          <label className={style.logo}>
+            WhiteDark<span style={{ color: '#302d2dfb' }}>.Net</span>
+          </label>
+        </NavLink>
+        <ul className={style.menu}>
+          <li className={style.item}>
+            <NavLink
+              className={(path === '/' && style.active) || undefined}
```

Comment:
```suggestion
              className={(path === '/' && style.active)}
```
if I'm not wrong you don't need undefined